rhino/src/main/java/org/mozilla/javascript/BaseFunction.java

```diff
@@ -31,7 +31,10 @@ static void init(Context cx, Scriptable scope, boolean sealed) {
         if (cx.getLanguageVersion() >= Context.VERSION_ES6) {
             obj.setStandardPropertyAttributes(READONLY | DONTENUM);
         }
-        obj.exportAsJSClass(MAX_PROTOTYPE_ID, scope, sealed);
+        IdFunctionObject constructor = obj.exportAsJSClass(MAX_PROTOTYPE_ID, scope, sealed);
+        if (cx.getLanguageVersion() >= Context.VERSION_ES6) {
+            ScriptRuntimeES6.addSymbolHasInstance(cx, scope, constructor);
```

Comment:
I was going to ask why you don't just call "defineProperty" here, but actually I'm wondering why we need this at all -- adding the new symbol case in the other places that you added it should be enough for this to work.